apis/v1beta1/httproute_types.go

```diff
@@ -720,8 +720,15 @@ type HTTPHeader struct {
 	Value string `json:"value"`
 }
 
-// HTTPHeaderFilter defines a filter that modifies the headers of an HTTP request
-// or response.
+// HTTPHeaderFilter defines a filter that modifies the headers of an HTTP
+// request or response. Implementations MUST NOT accept more than one action
+// for the same header name (including multiple actions of the same type). If
```

Comment:
This seems like something we should add to webhook validation?

Reply:
Yeah, probably better insofar as it's possible and we'd rather reject bad changes up front rather than let them go in and then have to invalidate a route that was previously working.